src/parser.cpp

```diff
@@ -49,6 +50,8 @@ std::unique_ptr<Stmt> Parser::stmt()
     {
     case Token::Type::VAR:
         return this->var_decl();
+    case Token::Type::IDENTIFIER:
```

Comment:
The following statement will be parsed as an assignment, even though it isn't one:

```
x + 3;
```

You may have to look one more character ahead for the `=` operator to determine if it is an assignment.